src/box/lua/stat.c

```diff
@@ -182,21 +195,44 @@ lbox_stat_net_call(struct lua_State *L)
 {
 	lua_newtable(L);
 	iproto_rmean_foreach(set_stat_item, L);
+	inject_current_stat(L, "CONNECTIONS", iproto_connection_count());
+	inject_current_stat(L, "REQUESTS", iproto_request_count());
+	return 1;
+}
 
-	lua_pushstring(L, "CONNECTIONS");
-	lua_rawget(L, -2);
-	lua_pushstring(L, "current");
-	lua_pushnumber(L, iproto_connection_count());
-	lua_rawset(L, -3);
-	lua_pop(L, 1);
+static int
+lbox_stat_net_thread_index(struct lua_State *L)
+{
+	size_t value;
+	const int thread_id = luaL_checkinteger(L, -1);
```

Comment:
Better set `thread_id` to `luaL_checkinteger(L, -1) - 1` here. Then you won't need to decrement 1 from thread_id below, and the variable name will be consistent with the thread_id definition.

Reply:
fixed